WebHandler/__init__.py

```diff
@@ -153,19 +159,31 @@ def get_total_no_of_cases_wrapper():
                             requests.post(
                                 url="https://ecourtsapiservice-dev.azurewebsites.net/api/WebHandler?method=advocatecasesbynamepagination", timeout=1, json=req_body)
                         except Exception as e:
+                            traceback.print_exc()
+                            logger.info(str(e), exc_info=True)
+                            end = datetime.datetime.now()
+                            total = end - start
                             requests.post(url=req_body["callBackUrl"], timeout=10, json={
-                                "error": str(e), "message": "Request Failed", "request": {"body": req_body, "params": req_params}})
+                                "error": str(e), "message": "Request Failed", "request": {"body": req_body, "params": req_params, "time": total}})

```

Comment:
in every callback request I want to include an error code
So here you can say 1, in another place you can say 2. Every exception will have different code.
So we will know which except block was called. 
if they are all same we dont know which except block was called.